base/applications/mscutils/servman/progress.c

```diff
@@ -147,6 +158,11 @@ unsigned int __stdcall ActionThread(void* Param)
                 /* We're done, slide the progress bar up to the top */
                 CompleteProgressBar(ProgressData->hProgress);
             }
+            else
+            {
+                   LoadStringW(hInstance, IDS_ERROROCCURED, szMessage, _countof(szMessage));
+                MessageBoxW(NULL, szMessage, NULL, MB_ICONERROR);
```

Comment:
Print error code as well?

Reply:
Error code not available yet. Will be implemeted when https://jira.reactos.org/browse/CORE-16950 will be adressed (on my ToDo list for a future PR)